templates/templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/_BenchmarkProjectName_.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup Condition="'$(consoleApp)' == 'true'">
-    <TargetFramework>netcoreapp3.0</TargetFramework>
-    <OutputType>Exe</OutputType>
```

Comment:
Without OutputType, we get the following error when `consoleApp` was selected during the project creation:

```sh
Unable to run your project.
Ensure you have a runnable project type and ensure 'dotnet run' supports this project.
A runnable project should target a runnable TFM (for instance, net5.0) and have OutputType 'Exe'.
The current OutputType is 'Library'.
```

Reply:
> Without OutputType

@am11 great catch, thank you! @ExceptionCaught could you please address that in a separate PR?

Reply:
hi @am11 , what was the dotnet version that you run `new` on? I did a few dry run on my local. with or without the `consoleapp`, could not see the error you mentioned.
![image](https://user-images.githubusercontent.com/11344974/103945321-23ec7300-5189-11eb-822b-118fc89df08d.png)
![image](https://user-images.githubusercontent.com/11344974/103945332-277ffa00-5189-11eb-9f2f-65fef5162b9d.png)

Reply:
The command which gives the error is 'dotnet run'.